update instructions for data and dashboards (#14)
hubverse-org/hub-dashboard-template: README.md

1. Add markdown content to `pages/`
2. Update `site-config.yml`

>     i. `hub` is the github slug for your active hub. This example defaults to

Reviewer: In all honesty some folks might not be familiar with the term `slug` (it's very web devy). Should we be more explicit? ie. would GitHub repository `owner/name` be better?

Author: Good point! It's definitely jargon. 

```suggestion
    i. `hub` is the github repository (`owner/repo`) for your active hub. This example defaults to
```

>        the CDC FluSight hub

Author: ```suggestion
       the CDC FluSight hub (`cdcepi/FluSight-forecast-hub`)
```

Reviewer: I still think it's best to explicitly mention we are talking about a repo's owner and name. It's something users will be familiar with while slug is not an intuitive description imho.

    ii. `title` is the title of your dashboard
    iii. `pages` is a list of pages you want included in the top bar after the
         home page (index.html) and forecasts (forecast.html).
3. Update `predtimechart-config.yml` according to the instructions at
   [hub_predtimechart](https://github.com/hubverse-org/hub-dashboard-predtimechart/tree/main?tab=readme-ov-file#required-hub-configuration).
4. (Optional) Add `predevals-config.yml` if you have oracle output that you can

>    use to generate predevals data. (See

Author: good catch!

```suggestion
   use to generate predevals data. (See below for a link to the documentation and
```

   [reichlab/flusight-dashboard](https://github.com/reichlab/flusight-dashboard/blob/main/predevals-config.yml)
   for an example).
5. (Optional) If your hub has an S3 bucket associated with it (you can find

>    this in the `hub-config/admin.json` file of your hub), you can add this

Author: ```suggestion
   this in the `cloud.host.name` property of the `hub-config/admin.json` file of your hub), and you are including a `data.qmd` page, you can add this
```
